### ISSUE TITLE
Unable to create/update JSON attributes

### ISSUE THREAD

**Issue by io238** [beyondcode/invoker-community#30]
I am unable to create/update JSON attributes (casted as array) in a model.

### Attribute cast in model definition
```
protected $casts = [
    'main_countries' => 'array',
];
```

### Migration
```
$table->json('main_countries')->nullable();
```

I've tested this in two different environments:

## Win 10 / PHP 7.4.7 / MariaDB 10.4.13

The Invoker validation fails and the validation message states that the expected input should be a string.
This validation fails for **any** input (valid/invalid JSON strings or null/empty inputs).

![image](https://user-images.githubusercontent.com/26689138/106593243-d9280600-6550-11eb-87d1-b74b560dad39.png)

## Ubuntu 18.04 / PHP 7.4.9 / MySQL 5.7.32

In this environment there is no validation error, but the update on the attribute is not persisted in the DB.

**Comment by vilhelmjosander** [comment]
I have a similar issue, but the table won't even load when there are jsonb-columns in a postgres database:

------
**OS**: darwin
**Invoker Version**: 2.0.1
**PHP Binary**: /usr/local/bin/php
**Route**: /project/index?class=App%5CModels%5CXXX&time=1612275417618&project=remote-69E8jzvoYESpE4mceomQMd

**Error**:
```
_HumbugBox7091da86deae\Doctrine\DBAL\Exception 

  Unknown database type jsonb requested, _HumbugBox7091da86deae\Doctrine\DBAL\Platforms\PostgreSqlPlatform may not support it.

  at phar:///tmp/invoker.phar/vendor/doctrine/dbal/lib/Doctrine/DBAL/Platforms/AbstractPlatform.php:179
    175▕             $this->initializeAllDoctrineTypeMappings();
    176▕         }
    177▕         $dbType = \strtolower($dbType);
    178▕         if (!isset($this->doctrineTypeMapping[$dbType])) {
  ➜ 179▕             throw new \_HumbugBox7091da86deae\Doctrine\DBAL\Exception('Unknown database type ' . $dbType . ' requested, ' . static::class . ' may not support it.');
    180▕         }
    181▕         return $this->doctrineTypeMapping[$dbType];
    182▕     }
    183▕     public function hasDoctrineTypeMappingFor($dbType)

      [2m+3 vendor frames [22m
  4   phar:///tmp/invoker.phar/src/Factories/AttributeCollection.php:35
      _HumbugBox7091da86deae\Doctrine\DBAL\Schema\AbstractSchemaManager::listTableColumns()

  5   phar:///tmp/invoker.phar/src/Factories/AttributeCollection.php:15
      _HumbugBox7091da86deae\Invoker\Factories\AttributeCollection::getColumns()
```

**Comment by valuu-io** [comment]
@mpociot please re-open. The problem is still not resolved after updating to 2.0.2.

![image](https://user-images.githubusercontent.com/57669509/106952012-d257e980-6730-11eb-8c14-85827ea4555d.png)

Here is how the JSON migration is implemented by the two environments that I've tested:
### Win 10 / PHP 7.4.7 / MariaDB 10.4.13
```
longtext	utf8mb4	utf8mb4_bin		
```

### Ubuntu 18.04 / PHP 7.4.9 / MySQL 5.7.32
```
json
```

On Ubuntu it does not throw any validation errors, but it does not persist any changes to the DB.

**Comment by io238** [comment]
@mpociot please re-open. The problem is still not resolved after updating to 2.0.2.

![image](https://user-images.githubusercontent.com/57669509/106952012-d257e980-6730-11eb-8c14-85827ea4555d.png)

Here is how the JSON migration is implemented by the two environments that I've tested:
### Win 10 / PHP 7.4.7 / MariaDB 10.4.13
```
longtext	utf8mb4	utf8mb4_bin		
```

### Ubuntu 18.04 / PHP 7.4.9 / MySQL 5.7.32
```
json
```

On Ubuntu it does not throw any validation errors, but it does not persist any changes to the DB.
